e2e/cmd/flags.go

```diff
@@ -57,5 +57,6 @@ func bindKeyCreateFlags(cmd *cobra.Command, cfg *key.UploadConfig) {
 }
 
 func bindPortalAdminFlags(flags *pflag.FlagSet, cfg *admin.PortalAdminConfig) {
-	flags.StringVar(&cfg.Chain, "chain", cfg.Chain, "pause a specific chain")
+	flags.StringVar(&cfg.Chain, "chain", cfg.Chain, "Run admin command on a specific chain")
```

Comment:
can also combine into single flag maybe? if `--chain=*` or `--chain=all`

Can then add a `GetChains(netconf.Network)` functions that returns the chain list given